helm/blazegraph/templates/deployment.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "blazegraph.fullname" . }}
+  labels:
+    app: {{ template "blazegraph.name" . }}
+    chart: {{ template "blazegraph.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
```

Comment:
Should this always be 1?  If so we likely should not give the option to change it.